pkg/controller/rpaasinstance/rpaasinstance_controller.go

```diff
@@ -570,6 +754,52 @@ func newNginx(instance *v1alpha1.RpaasInstance, plan *v1alpha1.RpaasPlan, config
 			Lifecycle:       instance.Spec.Lifecycle,
 		},
 	}
+
+	if plan.Spec.Config.CacheHeaterEnabled {
```

Comment:
You could reduce the indented block short-circuiting here:
```suggestion
	if !plan.Spec.Config.CacheHeaterEnabled {
		return n
	}
```